### PR TITLE
TIP-1041: Add ES6 support

### PR DESCRIPTION
**Pull request by anaelChardan**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,13 +7,17 @@ services:
     volumes:
       - './:/srv/pim'
     working_dir: '/srv/pim'
+    networks:
+      - 'akeneo'
 
   node:
-    image: 'node:10-slim'
+    image: 'akeneo/node:10'
```
Comment (damien-carcel) [inline docker-compose.yml]: No, this diff is wanted

File: docker-compose.yml [hunk 2]
```diff
@@ -7,13 +7,17 @@ services:
     volumes:
       - './:/srv/pim'
     working_dir: '/srv/pim'
+    networks:
```
Comment (damien-carcel) [inline docker-compose.yml]: There is no needs to declare a network on CE std, using the default one created by compose is enough.

File: docker-compose.yml (hunk 1 above)
Comment (damien-carcel) [reply docker-compose.yml]: There are no tests on standard, so no need to have a 1.4 GB docker image.

Comment (BitOne) [reply docker-compose.yml]: But how dev from integrators develops their JS front without using yarn and webpack?

File: docker-compose.yml (hunk 2 above)
Comment (BitOne) [reply docker-compose.yml]: It's declared in EE-standard and this reduce the amount of diff when diffing between dev and standard. And explicit is good ;)

Comment (damien-carcel) [reply docker-compose.yml]: OK then.

File: docker-compose.yml (hunk 1 above)
Comment (damien-carcel) [reply docker-compose.yml]: Yarn is provided by `node:10-slim`, and webpack is provided by `yarn install`

Comment (damien-carcel) [reply docker-compose.yml]: Our custom image adds "only" Chrome and Puppeteer, for tests. Everything needed for building the PIM is already there in the `node:10-slim` image.

**Pull request merged**